README.md

```diff
@@ -24,10 +23,16 @@ cheats menu:
 - invincible                    // Makes Link invincible (no hurtbox, but can still take fall damage and drown)
 - invincible enemies*           // Makes some enemies invincible (infinite health)
 - moon jump                     // Hold R+A to moon jump
+- door storage                  // Toggle door storage collision
 - super clawshot (TBD)          // Clawshot is long and can grab most things (not implemented yet)
 - super spinner (TBD)           // Spinner is very fast and can hover (not implemented yet)
 
 flags menu:
+- general flags                 // Can toggle various general flags
```

Comment:
These would look probably look better if instead of "Can toggle various general flags", they said "Toggle various general flags". That form is preferred here.